sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -338,6 +340,19 @@ class OoyalaSkinBridgeListener {
     this.core.handleControlsTouch();
   };
 
+  onPipToggle(e) {
+    Log.log("Received PiP Toggle: " + e.isPipActivated + e.isPipButtonVisible);
+    this.skin.setState({
+      isRootPipActivated: e.isPipActivated,
+      //isRootPipButtonVisible: e.isPipButtonVisible
+    });
+    if (e.isPipButtonVisible) {
```

Comment:
it means that `isRootPipButtonVisible` will never be set to `false`

Reply:
Thank you. Such logic of handling if similar to Obj-C, but differs from Swift (that I relied on).
I decided that visibility is not necessary for this event (`onPipToggle`) and deleted it.